### PR TITLE
Standalone

### PR DESCRIPTION
**Pull request by d-ivashchuk**

### CODE REVIEW

File: src/runner.ts [hunk 1]
```diff
@@ -1,39 +1,49 @@
...
 import { config, configure } from './config';
 import { sendResultToAPI } from './upload';
 import { sendInitToAPI } from './sendInit';
 
 (async () => {
   await configure();
+  const isGenerateOnlyMode = config.generateOnly;
   try {
-    if (config.setPendingStatusCheck) {
+    if (config.setPendingStatusCheck && !isGenerateOnlyMode) {
```
Comment (chriskalmar) [inline src/runner.ts:13]: how about using `config.generateOnly` here as well, as it is that way below too and gives more context?

**Review by chriskalmar — COMMENTED** [review]
looks good 🤘

File: src/runner.ts [hunk 2]
```diff
@@ -1,39 +1,49 @@
...
       await sendInitToAPI();
     }
+    if (isUpdateMode()) {
+      log(
+        'Running lost-pixel in update mode. Baseline screenshots will be updated',
```
Comment (chriskalmar) [inline src/runner.ts:18]: nice idea

File: src/runner.ts (hunk 1 above)
Comment (d-ivashchuk) [reply src/runner.ts]: [47f6701](https://github.com/lost-pixel/lost-pixel-action/pull/16/commits/47f67018fc9a171b06c4d36d057746b004b85507)

File: src/shots/shots.ts [hunk 3]
```diff
@@ -67,6 +67,13 @@ const takeScreenShot = async ({
     fullPage: fullScreenMode,
     animations: 'disabled',
   });
+  if (isUpdateMode()) {
+    await page.screenshot({
+      path: shotItem.filePathBaseline,
+      fullPage: fullScreenMode,
+      animations: 'disabled',
+    });
+  }
```
[the remark this answers is not in the document]
Comment (chriskalmar) [reply src/shots/shots.ts:76]: indeed, we might need a different approach here.
we still need to be able to create the diffs, even in `generateOnly` mode.
but creating the same images in `baseline` and `current` would have also no value.
I think the best would be to have a final step in the process, that would clean up the `baseline` and copy over all `current` images.
that way we would still get the diffs and an updated baseline.
wdyt?

**Review by chriskalmar — APPROVED** [review]

**Pull request merged**